dts/bindings/input/renesas,ra-ctsu-wheel.yaml

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) 2025 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+  Renesas RA Capacitive Touch Wheel
+
+  This node is a renesas,ra-ctsu grandchild node and applies filters and calculations
+  to detect an input event on a group which is the child of renesas,ra-ctsu.
+  For more information see input/renesas,ra-ctsu.yaml
+
+  Example:
+
+  #include <dt-bindings/input/input-event-codes.h>
+
+  &ctsu {
+    compatible = "renesas,ra-ctsu";
+
+    group1 {
+      ...
+      wheel {
+        compatible = "renesas,ra-ctsu-wheel";
+        elements = <0>, <3>, <2>, <1>;
+        threshold = <711>;
+        event-code = <INPUT_ABS_WHEEL>;
+        status = "okay";
```

Comment:
nodes are "okay" by default, no point specifying this in the example, this is designed to be instantiated on the board so this is redundant, same for the other ones